examples/singularity.yaml

```diff
@@ -31,6 +31,13 @@ provision:
     set -eux -o pipefail
     command -v singularity >/dev/null 2>&1 && exit 0
     dnf install -y singularity
+- mode: user
+  script: |
+    #!/bin/bash
```

Comment:
Please add comments to explain the reason

Reply:
Or can we just fix the mount flag of /tmp via /etc/fstab ?

Reply:
That would revert the decision of the distribution, though.

Reply:
Setting APPTAINER_TMPDIR seems better